.github/workflows/tiobe.yaml

```diff
@@ -3,6 +3,7 @@ name: TIOBE Quality Checks
 on:
   schedule:
     - cron:  '0 7 1 * *'
+  workflow_dispatch:
```

Comment:
unrelated change

Reply:
Yeah, I'm not sure what the story is with this, because main@HEAD has:

```yaml
on:
  workflow_dispatch:
  schedule:
    - cron:  '0 7 1 * *'
```

So I don't understand why there's no complaint about a conflict, or the `workflow_dispatch` line being both above and below (and I could just remove the below one, which I must have had in an earlier version in my fork). I'll try to figure out what's happening here.